website/vercel.json

```diff
@@ -955,6 +955,66 @@
     {
       "source": "/learning-and-resources/how-to-guides",
       "destination": "/connect-data/how-to-guides"
+    },
+    {
+      "source": "/advanced-concepts/embed-appsmith-into-existing-application1",
+      "destination": "/advanced-concepts/embed-appsmith-into-existing-application"
+    },
+    {
+      "source": "/datasource-reference/querying-snowflake",
+      "destination": "/connect-data/reference/querying-snowflake-db"
+    },
+    {
+      "source": "/assets/images/:path*",
+      "destination": "/"
+    },
+    {
+      "source": "/assets/files/:path*",
+      "destination": "/"
+    },
+    {
+      "source": "/getting-sta",
+      "destination": "/getting-started"
+    },
+    {
+      "source": "/advanced-co",
+      "destination": "/advanced-concepts"   
+    },
+    {
+      "source": "/core-concepts/application-layout",
+      "destination": "/core-concepts/building-ui/designing-an-application/application-layout"
+    },
+    {
+      "source": "/changelog",
+      "destination": "https://github.com/appsmithorg/appsmith/releases"
+    },
+    {
+      "source": "/getting-started/tutorials/review-moderator-dashboard/advanced-concepts/invite-users",
+      "destination": "/advanced-concepts/invite-users"
+    },
+    {
```

Comment:
can we just redirect to these tutorial links directly to the new tutorials?

Reply:
These are not tutorial links. Actually, due to a malformed cross-reference, the link appends the `invite users` reference. So this should take users to the invite users page.